app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -21,8 +21,8 @@
   </div>
   <div class="fw-bold fs-s mod-center-section-title resources">RESOURCES</div>
   <div id="mod-center-other-options">
-    <a href="#" class="crayons-link crayons-link--block">
-      Feedback?
+    <a href="/connect/<%= @community_mod_channel.slug %>" target="_blank" rel="noopener noreferrer nofollow" class="crayons-link crayons-link--block">
```

Comment:
Need an `if` statement around this anchor link

Reply:
🤦